js/phila-scripts.js

```diff
@@ -1,12 +1,58 @@
-/*! phila.gov-theme 0.11.5 phila-theme.js 2015-03-27 11:19:08 AM */
+/*! phila.gov-theme 0.16.10 phila-theme.js 2015-10-16 9:51:08 PM */
 new List("filter-list", {
     valueNames: [ "item", "item-desc" ]
 });
 
-function searchPhilaGov() {
-    var input = document.getElementById("search-form"), value = input ? input.value : "defaultText";
-    window.location.href = "https://www.google.com/#q=site:phila.gov+" + escape(value);
-}
+var browse = {
```

Comment:
Oh, this browse block. That isn't part of the search. Not sure how it sneaked into the PR.